<launch>
    <node pkg="imu_reader" type="imu_reader_sync_node" name="sync_imu" output="screen">
        <param name="imu_topic1" type="string" value= "/mynteye/imu/data_raw"/>    #imu topic的名字
        <param name="imu_name1" type="string" value= "mynteye_imu"/>
        <param name="imu_topic2" type="string" value= "/imu/data"/>    #imu topic的名字
        <param name="imu_name2" type="string" value= "novatel_imu"/>
        <param name="data_save_path" type="string" value= "/home/jyp/Projects/calibration/"/>
    </node>
</launch>

